Core/Dashboards/DashboardService.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Core.Dashboards.Contracts;
+using Core.Dashboards.Models;
+using Core.Sessions.Contracts;
+using FluentResults;
+using Microsoft.Extensions.Logging;
+
+namespace Core.Dashboards;
+
+public class DashboardService : IDashboardService
+{
+    private readonly ILogger<DashboardService> _logger;
+    private readonly IDashboardRepository _dashboardRepository;
+    private readonly ISessionRepository _sessionRepository;
+    public DashboardService(ILogger<DashboardService> logger, IDashboardRepository dashboardRepository, ISessionRepository sessionRepository)
+    {
+        _logger = logger;
+        _dashboardRepository = dashboardRepository;
+        _sessionRepository = sessionRepository;
+    }
+
+    public async Task<Result<GetExercisesInSessionCombinedInfo>> GetExercisesInTimedSession(int sessionId, int userId)
```

Comment:
Merge with the following function in accordance to the combination of dashboard endpoints.